### PR TITLE
Gift Entry: Edit Batch Information

### PR DESCRIPTION
**Pull request by kenlewis-sfdo**
As part of [W-038427](https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001HqJ2QAK/view) [Edit Batch in new HGE Batch Creation Wizard], the following architecture changes have been made to the gift entry component hierarchy: 

* Introduction of an Aura wrapper component [GE_GiftEntryForm] 
* Removal of geFormRenderer from geHome

The primary motivation for these changes is so that a modal dialog window can be displayed allowing a user to edit batch information while on the gift entry page.

More detail regarding these changes to the component hierarchy can be found here: https://salesforce.quip.com/4XLBADao50UZ

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

- GE_GiftEntryForm
- geGiftEntryFormApp (renamed from geBatchGiftEntryApp)

Labels:

- commonOpen
- geEditBatchInfo 

# Deleted Metadata

### CODE REVIEW

**Comment by kenlewis-sfdo** [comment]
**lurch: attach W-038427

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HqJ2QAK" target="_blank">W-038427</a>

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -5642,7 +5650,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Gift Entry Update</shortDescription>
+        <shortDescription>Update</shortDescription>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels]: @kenlewis-sfdo With this updated to a more general description, should (or can) we also update the fullName to be commonUpdate?

**Comment by melissabarber** [comment]
@kenlewis-sfdo @ptewson-sfdo I reviewed the custom labels and added one comment there. Other than that I'm good on this PR.

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (kenlewis-sfdo) [reply src/labels/CustomLabels.labels]: Thanks, @melissabarber !  Just updated this.

File: src/aura/GE_GiftEntryForm/GE_GiftEntryForm.cmp [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<aura:component
+        implements="force:hasSObjectName,force:hasRecordId,flexipage:availableForAllPageTypes"
```
[the remark this answers is not in the document]
Comment (kenlewis-sfdo) [reply src/aura/GE_GiftEntryForm/GE_GiftEntryForm.cmp]: Yep!  When loaded on the record home page, the platform populates these variables with those values.

**Pull request merged**